test/extensions/http/header_validators/envoy_default/http2_header_validator_test.cc

```diff
@@ -438,15 +438,6 @@ TEST_F(Http2HeaderValidatorTest, ValidateResponseHeaderMapEmptyGenericName) {
                              UhvResponseCodeDetail::get().EmptyHeaderName);
 }
 
-TEST_F(Http2HeaderValidatorTest, ValidateRequestTrailersAllowUnderscoreHeadersByDefault) {
```

Comment:
This test is identical to the https://github.com/envoyproxy/envoy/blob/a298895dd66a8df6ff834c5d646c38e0fc21e105/test/extensions/http/header_validators/envoy_default/http2_header_validator_test.cc#L729. So I removed this one.